main/lsp/lsp.cc

```diff
@@ -55,11 +55,6 @@ LSPLoop::setupLSPQueryByLoc(unique_ptr<core::GlobalState> gs, string_view uri, c
                          move(gs));
     }
 
-    if (errorIfFileIsUntyped && fref.data(*gs).strictLevel < core::StrictLevel::True) {
```

Comment:
WDYT about keeping this logic to have untyped files report a better error & to short-curcuit?

Reply:
Works for me. I removed it since, after this change, no feature used it. Hover didn't even use it; it just happened to not work in untyped files.